index.js

```diff
@@ -4,7 +4,8 @@ var hljs = require('highlight.js');
 // import hljs from 'highlight.js';
 
 var vueHighlightJS = {};
-vueHighlightJS.install = function install(Vue) {
+vueHighlightJS.install = function install(Vue, options) {
```

Comment:
Might need to add a default value, or handle the possibility of options not being sent.
```js
vueHighlightJS.install = function install(Vue, options = {}) {
```

Reply:
This would be good to add!